From da8e1f93194ca8c8e781131c008e7e1a216514b6 Mon Sep 17 00:00:00 2001
From: John Kacur <jkacur@redhat.com>
Date: Mon, 25 Nov 2013 15:54:27 +0100
Subject: [PATCH 01/15] docs: Remove stray </a>

Remove a stray </a> without a paired <a>. This was probably left over from
an early edit. It doesn't cause any harm for html parsers that ignore it,
but can cause complaints by tools that look for problems in xml.

Signed-off-by: John Kacur <jkacur@redhat.com>
Signed-off-by: Jiri Kastner <jkastner@redhat.com>
Signed-off-by: John Kacur <jkacur@redhat.com>
---
 docs/oscilloscope+tuna.html | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docs/oscilloscope+tuna.html b/docs/oscilloscope+tuna.html
index 9bba9ae43898..de3b4d7f7ae6 100644
--- a/docs/oscilloscope+tuna.html
+++ b/docs/oscilloscope+tuna.html
@@ -103,7 +103,7 @@ OSADL - The Open Source Automation Development Lab
 		    
 		<p class="news-single-author">By: Carsten Emde (Used by permission)</p>
 		<h3>New versatile tools are helping us to manage real-time challenges under Linux (but you still can't tuna fish).</h3>
-		<div class="news-single-img"><img src="uploads/pics/spike.gif" width="144" height="107" border="0" alt="" title="" /></a><p class="news-single-imgcaption">The "CyclictestoSCOPE" in action</p></div> 
+		<div class="news-single-img"><img src="uploads/pics/spike.gif" width="144" height="107" border="0" alt="" title="" /><p class="news-single-imgcaption">The "CyclictestoSCOPE" in action</p></div>
 		<p class="bodytext">Do you wish you had a built-in oscilloscope that continuously displays the latency of your system? Here is one - thanks to Arnaldo Carvalho de Melo who not only wrote the <strong>oscilloscope</strong> but also the graphical process inspection and modification tool <strong>tuna</strong> that is part of the same tool package. In addition, these tools are not only very versatile tools that help us to manage real-time challenges - they also are excellent examples of the power and the beauty of the Python script language and its libraries. <br /> </p>
 <p class="bodytext">However, before we can install and run these tools, we must learn a little bit about <strong>cyclictest</strong> and its command line arguments, since this is where oscilloscope gets its data from.</p>
 <h5>Cyclictest<br /></h5>
-- 
1.8.3.1

